microscopium/screens/cellomics.py

```diff
@@ -147,6 +147,9 @@ def snail_stitch(fns):
     ----------
     fns : list of array, shape (M, N)
         The list of 25 image files to be stitched together.
+    order : array of int, shape (O, P)
+        The order of the stitching, with each entry referring
+        to the index of
```

Comment:
Tune in next week for the exciting conclusion of the description of the "order" argument! =P